sherpa-onnx/c-api/c-api.cc

```diff
@@ -58,21 +58,51 @@ SherpaOnnxOnlineRecognizer *SherpaOnnxCreateOnlineRecognizer(
 
   recognizer_config.model_config.transducer.encoder =
       SHERPA_ONNX_OR(config->model_config.transducer.encoder, "");
+  recognizer_config.model_config.transducer.encoder_buf_begin =
+      SHERPA_ONNX_OR(config->model_config.transducer.encoder_buf_begin, nullptr);
```

Comment:
```suggestion
  recognizer_config.model_config.transducer.encoder_buf_begin =
      config->model_config.transducer.encoder_buf_begin;
```

And please change other places.